python/sentry_kafka_schemas/sentry_kafka_schemas.py

```diff
@@ -23,19 +24,25 @@ class SchemaNotFound(Exception):
     pass
 
 
-TopicSchema = TypedDict(
-    "TopicSchema",
-    {
-        "topic": str,
-        "version": int,
-        "type": Union[Literal["json"]],
-        "compatibility_mode": Union[Literal["none"], Literal["backward"]],
-        "resource": str,
-        "examples": Sequence[str],
-    },
-)
+class TopicSchema(TypedDict):
+    version: int
+    type: Literal["json"]
+    compatibility_mode: Literal["none", "backward"]
+    resource: str
+    examples: Sequence[str]
+
+
+class ServicesData(TypedDict):
+    consumers: Sequence[str]
+    producers: Sequence[str]
```

Comment:
Can we hardcode the literals for the repos we support?

Reply:
I have now hardcoded them in the test, because those type definitions are not actually enforced (we just cast to them)

I was also wondering whether we should generate the type definitions from a schema definition of topic.yml